### ISSUE TITLE
From 1.5.0 to 1.6.2, I can't find the function to update geoip.db and geosite.db.

### ISSUE THREAD

**Issue by ghkghkghk** [hiddify/hiddify-next#1051]
### Search first

- [X] I searched and no similar issues were found

### What Happened?

![image](https://github.com/user-attachments/assets/360c5414-2ddd-4451-b644-44af65adf109)

As the title says, how do I update geoip.db and geosite.db in the latest version (1.6.2 dev)?

### Minimal Reproducible Example (MRE)

![image](https://github.com/user-attachments/assets/cd22f61e-58c8-4aeb-9c32-1be46350929f)

### Version

1.6.2 dev

### Platform/OS

Windows

### Are you willing to submit a PR? If you know how to fix the bug.

- [ ] I'm willing to submit a PR (Thank you!)

**Comment by ChapayTv** [comment]
I am also interested in this question. At the moment, all the sites that I visit are proxied. How to solve the problem?

**Comment by papampi** [comment]
It uses new sing-box rules and auto updates them, no need to worry about them anymore.

**Comment by ghkghkghk** [comment]
> It uses new sing-box rules and auto updates them, no need to worry about them anymore. I think every 120 hours or at each start.
![image](https://github.com/user-attachments/assets/62fc717a-2c34-499e-a1bb-43acf7416d89)

As you described, these two "db" files will be updated by sing-box every 120 hours. Is this correct?

**Comment by ChapayTv** [comment]
> It uses new sing-box rules and auto updates them, no need to worry about them anymore. I think every 120 hours or at each start.

This obviously doesn't work because now all my sites open from the IP server. domains . ru are not filtered!

**Comment by papampi** [comment]
> As you described, these two "db" files will be updated by sing-box every 120 hours. Is this correct?

Nope, it reads directly from urls , you can check the urls it is using by openning the `current-config.json` file ... it has some urls to the  `.srs` files like this:

```
    "rule_set": [
      {
        "type": "remote",
        "tag": "geoip-ir",
        "format": "binary",
        "url": "https://raw.githubusercontent.com/hiddify/hiddify-geo/rule-set/country/geoip-ir.srs",
        "update_interval": "120h0m0s"
      },
      {
        "type": "remote",
        "tag": "geosite-ir",
        "format": "binary",
        "url": "https://raw.githubusercontent.com/hiddify/hiddify-geo/rule-set/country/geosite-ir.srs",
        "update_interval": "120h0m0s"
      },
      {
        "type": "remote",
        "tag": "geosite-ads",
        "format": "binary",
        "url": "https://raw.githubusercontent.com/hiddify/hiddify-geo/rule-set/block/geosite-category-ads-all.srs",
        "update_interval": "120h0m0s"
      },
      {
        "type": "remote",
        "tag": "geosite-malware",
        "format": "binary",
        "url": "https://raw.githubusercontent.com/hiddify/hiddify-geo/rule-set/block/geosite-malware.srs",
        "update_interval": "120h0m0s"
      },
      {
        "type": "remote",
        "tag": "geosite-phishing",
        "format": "binary",
        "url": "https://raw.githubusercontent.com/hiddify/hiddify-geo/rule-set/block/geosite-phishing.srs",
        "update_interval": "120h0m0s"
      },
      {
        "type": "remote",
        "tag": "geosite-cryptominers",
        "format": "binary",
        "url": "https://raw.githubusercontent.com/hiddify/hiddify-geo/rule-set/block/geosite-cryptominers.srs",
        "update_interval": "120h0m0s"
      },
```

**Comment by ghkghkghk** [comment]
> > As you described, these two "db" files will be updated by sing-box every 120 hours. Is this correct?
> 
> Nope, it reads directly from urls , you can check the urls it is using by openning the `current-config.json` file. It has some urls to the `.srs` files like this:
> 
> ```
>     "rule_set": [
>       {
>         "type": "remote",
>         "tag": "geoip-ru",
>         "format": "binary",
>         "url": "https://raw.githubusercontent.com/hiddify/hiddify-geo/rule-set/country/geoip-ru.srs",
>         "update_interval": "120h0m0s"
>       },
>       {
>         "type": "remote",
>         "tag": "geosite-ru",
>         "format": "binary",
>         "url": "https://raw.githubusercontent.com/hiddify/hiddify-geo/rule-set/country/geosite-ru.srs",
>         "update_interval": "120h0m0s"
>       },
>       {
>         "type": "remote",
>         "tag": "geosite-ads",
>         "format": "binary",
>         "url": "https://raw.githubusercontent.com/hiddify/hiddify-geo/rule-set/block/geosite-category-ads-all.srs",
>         "update_interval": "120h0m0s"
>       },
>       {
>         "type": "remote",
>         "tag": "geosite-malware",
>         "format": "binary",
>         "url": "https://raw.githubusercontent.com/hiddify/hiddify-geo/rule-set/block/geosite-malware.srs",
>         "update_interval": "120h0m0s"
>       },
>       {
>         "type": "remote",
>         "tag": "geosite-phishing",
>         "format": "binary",
>         "url": "https://raw.githubusercontent.com/hiddify/hiddify-geo/rule-set/block/geosite-phishing.srs",
>         "update_interval": "120h0m0s"
>       },
>       {
>         "type": "remote",
>         "tag": "geosite-cryptominers",
>         "format": "binary",
>         "url": "https://raw.githubusercontent.com/hiddify/hiddify-geo/rule-set/block/geosite-cryptominers.srs",
>         "update_interval": "120h0m0s"
>       },
> ...
> ```
> 
> The first two rules will be added by the region you select, And the rest will be added if you turn on block advertisement

So, manually deleting the two "db" files in the working directory will have no effect, right?
![image](https://github.com/user-attachments/assets/f4c9e86e-29a6-4d81-82a3-6d122d064b8c)



I would also like to ask, the size of this online "srs" file is very small, can it replace the function of the original db format file?
![image](https://github.com/user-attachments/assets/d45c2c54-8ba9-4e53-a453-7df4ded2963a)

**Comment by papampi** [comment]
> So, manually deleting the two "db" files in the working directory will have no effect, right?

You can clearout content of geo-assets folder with no problem in 1.6.x

> I would also like to ask, the size of this online "srs" file is very small, can it replace the function of the original db format file?

Usually so small, that's why sing-box changed it ruling format so it doesn't need to load a 10 MB .db file for 100 KB .db rules.
You can download the `.srs` files and check their size.
For example [geoip-ru.srs](https://raw.githubusercontent.com/hiddify/hiddify-geo/rule-set/country/geoip-ru.srs) is about 65 KB

**Comment by ChapayTv** [comment]
What should I do on Android?

**Comment by papampi** [comment]
> What should I do on Android?

Nothing ... just use it ...
If you dont want old .db files, you can clear out its data ... but not needed on any of them.
New hiddify uses the new sing-box rules

**Comment by ChapayTv** [comment]
> > What should I do on Android?
> 
> Nothing ... just use it ... If you dont want old .db files, you can clear out its data ... but not needed on any of them. New hiddify uses the new sing-box rules
So I used to have all .ru domains opened directly, but now with some kind of fig through a proxy. How do I do this?

**Comment by ChapayTv** [comment]
> > What should I do on Android?
> 
> Nothing ... just use it ... If you dont want old .db files, you can clear out its data ... but not needed on any of them. New hiddify uses the new sing-box rules

My routes are not filtered .ru

**Comment by papampi** [comment]
No idea ... Have you set correct region?
Maybe uninstall /install ... see how it goes. 
Or wait for stable release.

**Comment by ChapayTv** [comment]
> No idea ... Have you set correct region? Maybe uninstall /install ... see how it goes. Or wait for stable release.

I have a stable version, I deleted and reinstalled the program. It's no use.
